### DISCUSSION TITLE
Jupyter notebooks are not rendering anymore: "An error occurred"

### DISCUSSION THREAD

**Discussion by tcapelle** [community/community#9692] [Other Feature Feedback, Questions, & Ideas] [answered]
Notebooks from some of my repos are not rendering anymore, just saying 
<img width="541" alt="image" src="https://user-images.githubusercontent.com/18441985/148367439-467a184b-469b-46f5-b560-ae2da64fd402.png">
I tracked the issue to a very minimal example [here](https://github.com/tcapelle/nb_render) where two almost identical notebooks don't behave correctly.

The one that has a colon `:` in the title does not render, the one with out, renders...

**Comment by tcapelle** [comment, marked as answer]
This is solved
<img width="603" alt="image" src="https://user-images.githubusercontent.com/18441985/148527731-dd0f8d37-17d7-44c4-b214-dc16217ef085.png">

**Comment by 34-anish** [comment]
Got this issue again

**Reply by lamg** [reply]
I can confirm. This two unrelated notebooks show a similar image to the one above. [1](https://github.com/tcapelle/timeseries_fastai/blob/master/05_regression.ipynb) [2](https://github.com/lamg/structured_programming_in_fsharp/blob/master/bfs.ipynb)

**Reply by ndhquan11** [reply]
I just spend 3 hours to fix this, thought that sth wrong with nbformat :D

**Reply by talhaahussain** [reply]
Same issue here.

**Reply by IrisHinrichs** [reply]
I am also having this issue.

**Reply by Kazake95** [reply]
anyone found any solution?

**Reply by kembala** [reply]
same here

**Reply by kembala** [reply]
We have checked out notebook with the exact nbconvert and nbformat and it works locally.

**Reply by lamg** [reply]
Seems to be fixed now

**Reply by kembala** [reply]
indeed

**Comment by 34-anish** [comment]
![Uploading image.png…]()
